src/main/java/com/hillel/evoApp/model/ChatRoom.java

```diff
@@ -8,22 +8,28 @@
 import javax.persistence.*;
 import java.util.List;
 
-@EqualsAndHashCode(callSuper = true)
-@Entity
-@Table(name = "chat_rooms")
-@Data
-@NoArgsConstructor
-@AllArgsConstructor
-public class ChatRoom extends BaseEntity {
-
-    @Column(name = "name")
-    private String name;
-
-//    2019-09-21: jack.petrov: I need time to figured out why next line does not work.
-//    @ManyToMany(mappedBy = "chat_rooms", fetch = FetchType.LAZY)
+//    2019-11-21: jack.petrov: I need time to figured out correct mapping chats and messages
+//@EqualsAndHashCode(callSuper = true, exclude = {"messages", "sender", "receiver"})
+//@Entity
+//@Data
+//@NoArgsConstructor
+//@AllArgsConstructor
+//public class ChatRoom extends BaseEntity {
```

Comment:
Please completely remove this class. We will add it when we start our work on the chat-room.